logs/app.log

```diff
@@ -0,0 +1,72 @@
+2022-03-21 20:22:07 INFO  Director:27 - Programm start
```

Comment:
do not commit

Reply:
i added in ignoreFile. but it commited whatever